activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb

```diff
@@ -191,6 +195,10 @@ def index_algorithms
         }
       end
 
+      def return_value_after_insert?(column) # :nodoc:
+        column.auto_populated?
+      end
```

Comment:
Should we make this the default in `AbstractAdapter`?

Reply:
This method is `:nodoc:` and the same for all adapters, so I think yes. Did that.